CHANGELOG.md

```diff
@@ -9,10 +9,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 * AVIF_ENABLE_WERROR is set to OFF by default.
 * Fix wrong alpha plane deallocation when decoded tile pixel format does not
-  match reconstructed output image pixel format.
-* Fix identical chunk skipping optimization when writing animation data.
+  match reconstructed output image pixel format (b/320234262).
+* Fix identical chunk skipping optimization when writing animation data
+  (b/321189607 and b/321189607).
```

Comment:
@vrabaud @y-guyon I just noticed these two bug numbers are the same. I don't know how to fix this, depending on whether there is a second bug number, so I will let you take care of this.